### DISCUSSION TITLE
Built-in Clock App 🕔

### DISCUSSION THREAD

**Discussion by hrstwn** [elementary/triage#171] [Ideas]
I feel like we need a built-in clock app because it's a basic feature most people would use, especially for productivity. In my personal use case, I frequently need a timer and world clock feature, which both exist only if I install something like Gnome Clocks. Shouldn't elementary have it too?

**Comment by lenemter** [comment]
There is an app in AppCenter called "Hourglass" which lets you set alarms, timers and stopwatches. Maybe such application build-in into elementary OS is something to consider in the future.

**Reply by hrstwn** [reply]
Yes, I’m aware of the app. It’s a good alternative but lacks World Clock feature for me.
I feel like it’s necessary to have built-in, even a fork of Gnome’s or Hourglass would be enough.

**Reply by ryonakano** [reply]
This is the current maintainer of Hourglass and I've just started working on implementing the World Clock feature: https://github.com/sgpthomas/hourglass/pull/229
